### ISSUE TITLE
Syntax error on nonnull / nullable

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#86]
Example:

```
+(nonnull NSString *)extractSomethingFrom:(nonnull NSString *)something {
    ...
}
```

I'll get error "unexpected identifier".

It should be handled and in case `nullable`, the type should be converted to an optional.

**Comment by drkameleon** [comment]
Good point.

So basically this one (your example) should be:

```swift
        class func extractSomethingFrom(something: String) -> String {
	
	}
```

Right?

**Comment by charlieMonroe** [comment]
Yes. And for:

```
+(nullable NSString *)extractSomethingFrom:(nonnull NSString *)something {
    ...
}
```

should become (notice String?):

```
class func extractSomethingFrom(something: String) -> String? {

}
```

**Comment by drkameleon** [comment]
Yup. Got it ;)

**Comment by drkameleon** [comment]
Re: https://developer.apple.com/swift/blog/?id=25

**Comment by charlieMonroe** [comment]
Note though that this link contains updated Xcode 7 syntax - many source files, however, include older syntax which has `nonnull` and `nullable`, vs. `_Nonnull` and `_Nullable`.

**Comment by drkameleon** [comment]
Well, I just put it here for reference.

Plus, I don't think it says `nonnull` and `nullable` have been deprecated:

> You can use _Nullable and _Nonnull almost anywhere you can use the normal C const keyword, though of course they have to apply to a pointer type. However, in the common case there’s a much nicer way to write these annotations: within method declarations you can use the non-underscored forms nullable and nonnull immediately after an open parenthesis, as long as the type is a simple object or block pointer.

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.0.

-----

**Example:**
```objective-c
@implementation aClass

+(nonnull NSString *)extractSomethingFrom:(nonnull NSString *)something {
    NSLog(@"nonnull");
}

+(nullable NSString *)extractSomethingElseFrom:(nullable NSString *)something {
    NSLog(@"nullable");
}

@end
```

**Output:**
```swift
class aClass {

	class func extractSomethingFrom(something: String) -> String {
		print("nonnull")
	
	}
	
	class func extractSomethingElseFrom(something: String?) -> String? {
		print("nullable")
	
	}
	
}
```
